test/jdk/javax/swing/text/Caret/8163124/CaretFloatingPointAPITest.java

```diff
@@ -222,11 +222,11 @@ public void paint(Graphics g) {
 
             Graphics2D g2d = (Graphics2D) g;
             g2d.draw(new Line2D.Float(c, cy, c, cy + ch));
-            g2d.draw(new Line2D.Float(cx, cy, cx + cw, cy));
-            g2d.draw(new Line2D.Float(cx, cy + ch, cx + cw, cy + ch));
```

Comment:
>  Since the test is about placing of caret at same position between characters, a plain vertical caret is sufficient to test so removed the horizontal line rendering. 

I think we can still keep those lines, as it's kinda neat and presents a canonical caret look, and still works after applying the rest of the fix (I tried Linux and MacOS).

Reply:
No, it leaves some artifact on macos so I decided to remove those..